release.yml

```diff
@@ -0,0 +1,92 @@
+version: 2.3.2
+
+# JIRA Project where the release issue should be created
+issue:
+  project: ENTSBT
+
+
+# If already created, provide the issue number of the Release, otherwise keep it empty.
+# It will be used by the command create-component to link the stakeholder issues
+#key:         # SB-1484
+
+schedule:
+  # Release date scheduled. It will also be used to populate the description to be created for each Jira stakeholder: component or starter owner
+  release: 2020-09-14
+  # Due date for the component/starter owner to release their new components, QE signoff it
+  due: 2020-08-18
+  # EOL of the Snowdrop release
+  eol: 2021-09-30
+
+components:
+  # List of JIRA Projects contributing to a Snowdrop release
+  # They will be used to create a component or starter
+  - name: Hibernate / Hibernate Validator / Undertow
```

Comment:
Does anybody know how should the Narayana and Vert.x versions be handled? We include the starters but do not include the underlying component versions in the same way as we do Hibernate, Infinispan etc. Is it OK since we do the starter prod ourselves or is this an issue?

Reply:
Good point. We probably need to create components for these dependencies as well since we need the product team to tell us which version we're supposed to be using. Do you have the associated JIRA project information for these products?

Reply:
JBTM for Narayana and I don't know for Vert.x.
Do they need to tell us a productised/upstream version that we should use or a version that they guarantee to work with Spring Boot (because the later they will not know).

Reply:
If I'm correct in the assumption that we're the ones making sure that a given version works with a given Spring Boot version, they only need to tell us which productized version we should use.

Reply:
Yes, we're the ones maintaining these starters and the project teams are not involved in Spring Boot integration.

Reply:
So should we create a ticket for these products as well to ask them which productized version we're supposed to use? Or are we going to handle this manually?

Reply:
So far we've been doing it manually without issues but one exception (which was resolved and shouldn't happen again). I don't think there is much need for it.
However, if the idea of this process change is to create issues for all the third party components and provide a trace record, it might make sense. In such case the issue might have to be different from the other projects though.